app/controllers/api/v1/study_search_results_objects.rb

```diff
@@ -61,7 +62,7 @@ def study_response_obj(study)
           end
         else
           study_obj = {
-            tdr_result: true,
+            study_source: 'TDR',
```

Comment:
switched to a string `study_source` (if anybody has a better name tho totally open) so that in case ever in the future there are other sorts of studies we'd be ready rather than just being TDR specific

Reply:
Good idea! The latest Emerald Empire update mentioned that they're ingesting LungMap data, which we're told may have cell-level metadata so it'd be good to be prepared for more than one study_source :)